### PR TITLE
Feature/course color extension

### PR DESCRIPTION
**Pull request by beatrisbu**
### Checklist
- [x] I tested *all* changes and *all* related features on the test server https://artemistest.ase.in.tum.de.

### Motivation and Context

Additions regarding the courses' colors. 

### Description

Added a column visualizing the color of each course (where the color is set) at the beginning of each course row in the Course Administration.
Added multiple colors to the list of default colors that are displayed in the window with colors when creating a new course.  
Adapted the height of the window.

### Steps for Testing

1. Log in to Artemis
2. Navigate to Course Administration
3. At the beginning of each course row you should now be able to see the course's color
4. Click on create new course 
5. In the Color field you should be able to see the new colors

### Screenshots 

Create new course

Course administration

### CODE REVIEW

**Comment by beatrisbu** [comment]
The TypeScript Formatting test is failing. I got feedback from the team that they are aware of a problem with the build, so this is most probably not a problem caused by my code. I do not get any formatting errors locally.

**Comment by krusche** [comment]
There is a small Typescript code formatting error.


22-Oct-2019 17:02:31 | yarn run v1.19.0
-- | --
22-Oct-2019 17:02:31 | $ prettier --check 'src/{main/webapp,test}/**/*.{json,ts,css,scss,html}'
22-Oct-2019 17:02:31 | Checking formatting...
22-Oct-2019 17:02:43 | src/main/webapp/app/components/color-selector/color-selector.component.ts
22-Oct-2019 17:02:46 | src/main/webapp/app/entities/course/course.component.html
22-Oct-2019 17:03:38 | Code style issues found in the above file(s). Forgot to run Prettier?


Please fix it

**Comment by krusche** [comment]
And could you also post a screenshot here?

File: src/main/webapp/app/entities/course/course.component.html [hunk 1]
```diff
@@ -55,7 +57,10 @@ <h4 id="course-page-heading" jhiTranslate="artemisApp.course.home.title">Courses
                 <tr *ngFor="let course of courses | sortBy: predicate:reverse; trackBy: trackId">
                     <ng-container *ngIf="!showOnlyActive || !course.endDate || (course.endDate | amIsAfter: today:'second')">
                         <td>
-                            <a [routerLink]="['/course', course.id, 'view']">{{ course.id }}</a>
+                            <div [ngStyle]="{ 'backgroundColor': course.color, 'width': '15px', 'height': '20px' }"> &nbsp;</div>
```
Comment (Minh13) [inline src/main/webapp/app/entities/course/course.component.html]: Hi, I applied prettier locally and got diffs in line 60 and 63, perhaps that's whats breaking your build

**Pull request merged**